code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -9,6 +9,7 @@
 	faction = list("ashwalker")
 	health = 200
 	maxHealth = 200
+	anchored = TRUE
```

Comment:
this is already inherited from this is inherited from `/mob/living/simple_animal/hostile/spawner`, and would also be inherited from `/mob/living/simple_animal/hostile/spawner/lavaland` with your existing change.